packages/gateway/test/utils.js

```diff
@@ -0,0 +1,24 @@
+import { Miniflare } from 'miniflare'
+
+export function getMf() {
```

Comment:
Please can we call this `getMiniflare` instead? I'm ok with `mf` for a local variable...